core/src/main/java/com/bettercloud/bigtable/orm/Dao.java

```diff
@@ -1,5 +1,7 @@
 package com.bettercloud.bigtable.orm;
 
+import org.apache.hadoop.hbase.client.Scan;
```

Comment:
The HBase client import in the Dao interface is a red flag to me. The interface should make no assumptions about the underlying implementation. I would prefer if the `scan` method exposed something more intuitive, like a "start" and "end" Key.